src/test/java/tigase/server/xmppclient/StreamManagementIOProcessorTest.java

```diff
@@ -197,4 +199,37 @@ public void testForceDelayForResumption() {
 				.forEach((OutQueue.Entry e) -> assertFalse(
 						e.getPacketWithStamp().isXMLNSStaticStr(new String[]{"iq", "delay"}, DELAY_XMLNS)));
 	}
+
+	@Test
+	public void testShouldRequestAck()
+			throws NoSuchFieldException, IllegalAccessException, TigaseStringprepException, InterruptedException {
+		StreamManagementIOProcessor processor = new StreamManagementIOProcessor();
+		Field ackRefMinDelayField = StreamManagementIOProcessor.class.getDeclaredField("ack_request_min_delay");
```

Comment:
Can't we simply make the field protected? It's a config field all in all

Reply:
we could create getter and setter as well. I've used this method as I do not see a point to expose this field to the other classes.